### PR TITLE
CrUX doc updates

### PR DESCRIPTION
**Pull request by tunetheweb**
Changes proposed in this pull request:

- Updates all references to old CrUX docs (https://developers.google.com/web/tools/chrome-user-experience-report) to new docs (https://developer.chrome.com/docs/crux/).
- Updates dates on docs - where the doc was up to date (I've left some of the translations without updating the date as there are other changes that have not yet been translated and would like to avoid giving the impression the translations are up to date when they are not).
- Removed references to CrUX API explorer that now no longer exists.
- Added some `chrome-ux-report` tags where appropriate.

### CODE REVIEW

File: src/site/content/en/fast/chrome-ux-report-bigquery/index.md [hunk 1]
```diff
@@ -8,16 +8,18 @@ description: |
...
 ---
 
 The raw data of the Chrome UX Report
-([CrUX](https://developers.google.com/web/tools/chrome-user-experience-report/))
+([CrUX](https://developer.chrome.com/docs/crux/))
 is available on
 [BigQuery](https://console.cloud.google.com/bigquery?p=chrome-ux-report), a
 database on the Google Cloud Platform (GCP). Using BigQuery requires a
-[GCP project](https://developers.google.com/web/tools/chrome-user-experience-report/getting-started#getting-started)
+[GCP project](https://console.cloud.google.com/bigquery?p=chrome-ux-report&d=all&page=dataset)
```
Comment (rviscomi) [inline src/site/content/en/fast/chrome-ux-report-bigquery/index.md:22]: Not sure what else to put here in lieu of a more detailed CrUX-specific walkthrough

```suggestion
[GCP project](https://cloud.google.com/free)
```

File: src/site/content/en/fast/chrome-ux-report/index.md [hunk 2]
```diff
@@ -8,17 +8,19 @@ description: |
...
+  - chrome-ux-report
 ---
 
-The [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report/) (informally known as CrUX) is a public dataset of real user experience data on millions of websites. Unlike lab data, CrUX data actually comes from [opted-in users](https://developers.google.com/web/tools/chrome-user-experience-report/#methodology) in the field. It measures the [Core Web Vitals](/vitals/) metrics, including Largest Contentful Paint (LCP), First Input Delay (FID), and Cumulative Layout Shift (CLS), as well as diagnostic metrics like Time to First Byte (TTFB) and First Contentful Paint (FCP).
+The [Chrome UX Report](https://developer.chrome.com/docs/crux/) (informally known as CrUX) is a public dataset of real user experience data on millions of websites. Unlike lab data, CrUX data actually comes from [opted-in users](https://developer.chrome.com/docs/crux/methodology/) in the field. It measures the [Core Web Vitals](/vitals/) metrics, including Largest Contentful Paint (LCP), First Input Delay (FID), and Cumulative Layout Shift (CLS), as well as diagnostic metrics like Time to First Byte (TTFB) and First Contentful Paint (FCP).
```
Comment (rviscomi) [inline src/site/content/en/fast/chrome-ux-report/index.md:17]: Deep link to User

```suggestion
The [Chrome UX Report](https://developer.chrome.com/docs/crux/) (informally known as CrUX) is a public dataset of real user experience data on millions of websites. Unlike lab data, CrUX data actually comes from [opted-in users](https://developer.chrome.com/docs/crux/methodology/#user-eligibility) in the field. It measures the [Core Web Vitals](/vitals/) metrics, including Largest Contentful Paint (LCP), First Input Delay (FID), and Cumulative Layout Shift (CLS), as well as diagnostic metrics like Time to First Byte (TTFB) and First Contentful Paint (FCP).
```

**Review by rviscomi — CHANGES_REQUESTED** [review]
Wow thank you for going through all of the existing content beyond just the CrUX guides! Great to get everything updated in one pass.

Left a few comments on the en versions, which should also apply to the translated versions.

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 3]
```diff
@@ -151,7 +152,7 @@ will return the "old CLS" metric:
 ```
 
 After June 8, the following
-[CrUX BigQuery](https://developers.google.com/web/tools/chrome-user-experience-report/bigquery/getting-started)
+[CrUX BigQuery](/chrome-ux-report-bigquery/)
```
[the remark this answers is not in the document]
Comment (tunetheweb) [reply src/site/content/en/blog/cls-web-tooling/index.md:155]: Done. I updated any "getting-started" links to the guide. But agree this is probably better linked to the official docs.

File: src/site/content/en/blog/vitals-spa-faq/index.md [hunk 4]
```diff
@@ -129,7 +129,7 @@ page](https://en.wikipedia.org/wiki/Landing_page).
 You can check your site's score for different aggregation methods using
 [PageSpeed Insights](https://pagespeed.web.dev/) or
 the [Chrome User Experience Report
-API](https://developers.google.com/web/tools/chrome-user-experience-report/api/guides/getting-started),
+API](/chrome-ux-report-api/),
```
[the remark this answers is not in the document]
Comment (tunetheweb) [reply src/site/content/en/blog/vitals-spa-faq/index.md:132]: Not so convinced on this one, but could go either way. There's already a link to the official docs further up the page btw, but the PSI link here is to the tool itself so maybe guide is better link here? WDYT?

File: src/site/content/en/fast/chrome-ux-report-bigquery/index.md (hunk 1 above)
Comment (tunetheweb) [reply src/site/content/en/fast/chrome-ux-report-bigquery/index.md:22]: Yeah I struggled with this one myself. Maybe a mind in hindsight and we should expand this page with the detail from that link before it's gone? Happy to do that in this PR if you agree.

If not, then would prefer the link I suggested that the generic free one you suggested.

Comment (tunetheweb) [reply src/site/content/en/fast/chrome-ux-report-bigquery/index.md:22]: I've opened a PR for the DCC docs to add this missing info: https://github.com/GoogleChrome/developer.chrome.com/pull/3250

Will update this PR to point to that.

**Comment by tunetheweb** [comment]
OK setting this back to Ready to Review @malchata @rviscomi since https://github.com/GoogleChrome/developer.chrome.com/pull/3250 has now been merged.

There's also a Percy failure but it's an intended change. I don't have permissions to approve it though. Is that something you need to do @malchata ?

**Review by malchata — APPROVED** [review]
After a cursory look to check links and such, this LGTM. Any deeper issues should be discussed with @rviscomi.

**Pull request merged**